ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -287,8 +287,10 @@
       msg: "Looking at oc cluster up command:  '{{ oc_cluster_up_cmd }}'"
 
   - name: Run oc cluster up to start the cluster
-    shell: "{{ oc_cluster_up_cmd }}"
+    shell: "{{ oc_cmd }} cluster down && {{ oc_cluster_up_cmd }}"
```

Comment:
It is the same command the first time as it is on retry. As far as I know there is no way to implement a different command for a retry. There are rescues with blocks but I don't think this will work quite the way I/we want either. The cluster should be in a down state before we get here if everything else has gone well so the first oc cluster down is basically instantaneous.